ktlint/build.gradle

```diff
@@ -44,3 +44,20 @@ dependencies {
   testImplementation deps.junit
   testImplementation deps.assertj
 }
+
+tasks.create("shadowJarExecutable") {
```

Comment:
I think it's better to use `register` here if there are no obstacles, according to [task configuration avoidance guide](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html#sec:how_do_i_defer_creation). Cause in case of `create`, the task will be created and configured eagerly.

Reply:
You are totally right, I am always confusing what to use to create a task :+1:

Reply:
Fixed.

Reply:
Gradle things :D